src/boot/efi/boot.c

```diff
@@ -2204,7 +2202,7 @@ static VOID config_load_xbootldr(
                                 continue;
 
                         /* Calculate CRC of entries array, too */
-                        c = ~crc32(UINT32_MAX, entries, sz);
+                        r = BS->CalculateCrc32(&entries, sz, &c);
```

Comment:
Not familiar with that, and I cannot find the manual of the function, but is it not necessary to check the return value `r`?

Reply:
You're right, fixed it.